distributed/scheduler.py

```diff
@@ -314,124 +342,110 @@ def _to_dict(self, *, exclude: "Container[str]" = ()) -> dict:
 
 
 class WorkerState:
-    """
-    A simple object holding information about a worker.
-
-    .. attribute:: address: str
-
-       This worker's unique key.  This can be its connected address
-       (such as ``'tcp://127.0.0.1:8891'``) or an alias (such as ``'alice'``).
+    """A simple object holding information about a worker."""
```

Comment:
Hrm, we're losing most of the docstring here, which I'm a little sad about (although not too sad).  I guess the reason not to have a solid docstring here is because then we're keeping two copies of the attributes and types around (and honestly we're probably going to struggle to do even that).  I guess that this makes sense.

Reply:
The Sphinx documentation is still fully generated for the attributes:
https://distributed--6132.org.readthedocs.build/en/6132/scheduling-state.html#worker-state

I found many examples of the docstring falling out of sync with the attributes - I find keeping the documentation of the attribute together with its annotation much more robust. It also lets you spot on the fly the undocumented ones.